app/models/cul/hydra/datastreams/encoded_text_datastream.rb

```
@@ -2,11 +2,13 @@ module Cul
 module Hydra
 module Datastreams
 class EncodedTextDatastream < ::ActiveFedora::Datastream
-  DEFAULT_PRIORITIES = [ Encoding::UTF_8, Encoding::ISO_8859_1, Encoding::WINDOWS_1252 ]
+  DEFAULT_PRIORITIES = [ Encoding::UTF_8, Encoding::WINDOWS_1252, Encoding::ISO_8859_1 ]
```

Putting cp1252 earlier in the list, since there's overlap with latin-1 but it will fail faster when checking validity